app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -271,7 +272,7 @@ export function determinePostReplaceSelection({
 export function isAppleDevice() {
   // IE has no DOMNodeInserted so can not get this hack despite saying it is like iPhone
   // This will apply hack on all iDevices
-  const caps = Discourse.__container__.lookup("capabilities:main");
+  let caps = helperContext().capabilities;
```

Comment:
Does this need to switch from `const` to `let`?

Reply:
No it doesn't. `let` vs `const` is not really a big deal and IMO is mostly a preference. I used to be bullish on it, making everything const, but eventually I came around when I realised that almost no bugs are prevented. It doesn't ensure immutability of the object, only re-assignment. And JS doesn't make any optimizations based on it currently.